packages/app/src/components/ChatPanel.module.css

```
@@ -0,0 +1,326 @@
+
+:global([data-color-scheme="dark"]) .title {
+  color: #999;
+}
+
+:global([data-color-scheme="dark"]) .modelSelect {
+  background: #2a2a2a;
+  color: #ccc;
+  border-color: #444;
+}
+
+:global([data-color-scheme="dark"]) .iconBtn {
```

Stylelint (17.10.0)

[error] Unknown pseudo-class selector ":global" (selector-pseudo-class-no-unknown)

(selector-pseudo-class-no-unknown)